### PR TITLE
Fix Darkmode colors : Radio Button, CheckBox, ToggleSwitch, ComboBox

### PR DESCRIPTION
**Pull request by MaximeDion-Work**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Bugfix

## Description

Colors were off when looking at the app in dark mode for the Radio Button, CheckBox, ToggleSwitch and ComboBox samples.

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [x] Tested iOS
- [x] Tested Android
- [ ] Tested WASM
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

**Comment by NicolasChampagne** [comment]
reword commit as fix(darkmode): ...

File: src/library/Uno.Material/Styles/Application/SelectionControlColors.xaml [hunk 1]
```diff
@@ -13,35 +13,35 @@
...
-					 Color="{StaticResource OnSurfaceColor}"
+					 Color="{ThemeResource OnSurfaceColor}"
 					 Opacity="0.08" />
 	<!-- TODO: This is the OnSurfaceDisabledBrush, remove when brush issue #41 is fixed -->
 	<SolidColorBrush x:Key="SelectControlToggleOffBackgroundColor"
 					 Color="#61000000" />
 	<!-- TODO: This is the OnPrimaryMediumBrush, remove when brush issue #41 is fixed -->
 	<SolidColorBrush x:Key="SelectControlToggleOnDisabledButtonColor"
-					 Color="#B6A8FB" />
+					 Color="{ThemeResource PrimaryVariantLightColor}" />
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Application/SelectionControlColors.xaml:44]: this doesn't fit with the comment. Can you validate with @NicolasChampagne  ?

File: src/library/Uno.Material/Styles/Controls/ComboBox.xaml [hunk 2]
```diff
@@ -27,16 +27,16 @@
...
 
-	<StaticResource x:Key="ComboBoxFocusedBackgroundThemeBrush"
-					ResourceKey="OnSecondaryMediumBrush" />
+	<SolidColorBrush x:Key="ComboBoxPlaceholderTextForegroundThemeBrush"
+					 Color="{ThemeResource OnBackgroundColor}"
+					 Opacity="0.74" />
 
-	<StaticResource x:Key="ComboBoxPlaceholderTextForegroundThemeBrush"
-					ResourceKey="OnSecondaryMediumBrush" />
+	<CornerRadius x:Key="BorderRadius">4</CornerRadius>
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Controls/ComboBox.xaml:39]: That's a too general name. Use the VariableNames in the zeplin.
https://app.zeplin.io/project/5e344ca7405a53a6ef303bc1/screen/5e4d88eb8a67138239e14783

File: src/library/Uno.Material/Styles/Application/SelectionControlColors.xaml (hunk 1 above)
Comment (NicolasChampagne) [reply src/library/Uno.Material/Styles/Application/SelectionControlColors.xaml:44]: true, possibly OnPrimaryMediumAlternativeLightBrush until the issue fix is merged

File: src/library/Uno.Material/Styles/Controls/ComboBox.xaml (hunk 2 above)
Comment (MaximeDion-Work) [reply src/library/Uno.Material/Styles/Controls/ComboBox.xaml]: Now it's DefaultComboBoxBorderRadius.

File: src/library/Uno.Material/Styles/Controls/ComboBox.xaml [hunk 3]
```diff
@@ -285,7 +286,7 @@
 						<!-- Elevated View -->
 						<toolkit:ElevatedView Background="{ThemeResource ComboBoxBackgroundColorBrush}"
 											  HorizontalAlignment="Stretch"
-											  CornerRadius="4"
+											  CornerRadius="{StaticResource DefaultComboBoxBorderRadius}"
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Controls/ComboBox.xaml:289]: Shouldn't the variable name start with `ComboBox` like on all variables on zeplin?

`ComboBoxBorderCorderRadius` ?

Comment (MaximeDion-Work) [reply src/library/Uno.Material/Styles/Controls/ComboBox.xaml]: Done

**Pull request merged**